csharp-refactoring/RefactoringObjectOrientationAbusers/RefactoringObjectOrientationAbusers/TemporaryField/Incorrect/BookstoreCustomer.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace RefactoringObjectOrientationAbusers.TemporaryField.Incorrect
+{
+    public class BookstoreCustomer
+    {
+        public int DiscountThreshold { get; set; }
+        public int BoughtBooksCount { get; set; }
+        public ICollection<string> BoughtBooks { get; set; }
```

Comment:
Update the article to say properties instead of fields 
![image](https://user-images.githubusercontent.com/11241957/225846718-73c139d0-22ab-4361-a798-8aa0305e2469.png)

Reply:
I think it's the other way around, fields instead of properties, right? :)

Reply:
yes, indeed. My bad !